src/pages/home/report/ReportActionCompose/AttachmentPickerWithMenuItems.js

```diff
@@ -84,6 +87,9 @@ const propTypes = {
         // eslint-disable-next-line react/forbid-prop-types
         current: PropTypes.object,
     }).isRequired,
+
+    /** Whether or not the navigation is focused */
```

Comment:
```suggestion
    /** Whether navigation is focused */
```

Just so it's consistent with the rest of the app